packages/atlas-react/rollup.config.js

```diff
@@ -18,13 +20,16 @@ export default [
       scss({
         prefix: `@import "./src/common/styles/atlas.scss";`,
         output: "dist/atlas.css",
```

Comment:
the inline import is not necessary because we import this globally, to avoid duplicates.

Reply:
however, I will change this.  let's create a separate `@productplan/atlas-styles` package to manage these mixins - instead.